train.py

```diff
@@ -161,6 +161,8 @@
         n_eval_envs=args.n_eval_envs,
     )
 
+    print("Default hyperparameters for environment (ones being tuned will be overridden):")
```

Comment:
please put the print next to the correct print

Reply:
I just looked into this again to create a combined PR, and I apologize but I appear to be missing the problem with where the print statement is.

Reply:
I meant before this print: https://github.com/DLR-RM/rl-baselines3-zoo/blob/f6b3ff70b13d2c2156b3e0faf9994c107c649c82/utils/exp_manager.py#L252